src/app/components/Search/Search.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
```

Comment:
I'm not sure if this is the best place to put this. You might be able to make this more generic if it was outside of this component but I also don't know exactly how this function works.

Reply:
There might be a better home for this at the Application level perhaps. Not sure exactly what best practices are for Redux based React apps, but can certainly work on refactoring more as more experience is gleaned.

Reply:
Leave it for now. We can update later as we learn more.